src/interface/api-inbound-template.yaml

```diff
@@ -85,4 +85,142 @@ paths:
           $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/503'
       operationId: HelloGet
       summary: example get
-      description: The HTTP request GET /hello is used to return some example json.
\ No newline at end of file
+      description: The HTTP request GET /hello is used to return some example json.
+  /thirdpartyRequests/transactions:
+    #parameters:
```

Comment:
Should these be commented out?

Reply:
For now, yes. There is some header middleware/plugin that needs to be implemented.

Reply:
let add TODO why we want to keep this commented code here

Reply:
done.